src/test/java/au/org/emii/GenerationIT.java

```diff
@@ -104,11 +102,9 @@ public void soop_sst_trajectory_IT() throws Exception {
 	}
 
 
-
-
-	// test zip streaming of data...
+	// test zip streaming of data using builder...
 	@Test
-	public void anmn_timeseries2_IT() throws Exception 
+	public void anmn_timeseries2_IT() throws Exception
 	{
 		System.out.println( "**** anmn timeseries ****" );
```

Comment:
Do we need this.  You can always look at the report.   An IDE will display this info for you.